### ISSUE TITLE
Late ferries are shown as on time

### ISSUE THREAD

**Issue by mileserickson** [anstosa/ferry.fyi#69]
Today, 2022-06-08, both vessels are exceptionally late, but ferry.fyi is showing the normal
schedule.

![85F34327-25A7-40B5-A0BC-2C2E72C95F6B](https://user-images.githubusercontent.com/68300/172743185-f1710b59-ff50-40ac-8e5f-5221d9fd74b7.png)
![F13C8D05-E953-49B8-88E3-7ABB076D7EF4](https://user-images.githubusercontent.com/68300/172743191-7e0e03a8-97b3-4c54-a3e4-d2677f68824a.png)

**Comment by anstosa** [comment]
Yeah, looks like they pushed a bulletin for it but didn't actually update the delay numbers in the API. I've noticed this happens a lot and I want to start calculating my own delay times using the GPS coordinates instead of trusting the published numbers

**Comment by anstosa** [comment]
GPS delay implemented
